service/kas/kas.proto

```diff
@@ -31,6 +31,38 @@ message LegacyPublicKeyRequest {
   string algorithm = 1;
 }
 
+message KeyAccess {
+  string encrypted_metadata = 1;
+  bytes policy_binding = 2;
+  string protocol = 3;
+  string key_type = 4;
+  string kas_url = 5;
+  string kid = 6;
+  string split_id = 7;
```

Comment:
This field and several others are ignored by KAS at the moment, but we may want to model them properly anyway.

```suggestion
  string split_id = 7 [json_name = "sid"];
```